lib/src/builder/builder.dart

```diff
@@ -253,6 +253,17 @@ class OverReactBuilder extends Builder {
     return null;
   }
 
+  static const _ignoreForFileCodes = {
+    // Needed when the component itself or some of its props are deprecated,
+    // or if deprecated types are referenced within the boilerplate.
+    'deprecated_member_use_from_same_package',
+    // These two are needed for the `?? null` workaround in prop accessors.
+    'unnecessary_null_in_if_null_operators',
+    'prefer_null_aware_operators',
+    // Needed to ignore the `requiredPropNamesToSkipValidation` checks.
+    'invalid_use_of_visible_for_overriding_member',
```

Comment:
I added this ignore to fix my test setup, which validated there were no warnings in generated code